person.rb

```diff
@@ -1,9 +1,11 @@
+require_relative 'nameable'
+
 class Person
+  include Nameable
```

Comment:
- [ ] @jkanyi-web Great work adding the correct_name method in the person class, however, this class is supposed to inherit from the `Nameable` class.
![image](https://github.com/jkanyi-web/OOP_School_Library/assets/61408860/7bcea5c1-94fe-4cfa-8c8c-505395359324)